pisa/utils/config_parser.py

```diff
@@ -614,6 +617,26 @@ def parse_pipeline_config(config):
     # Get and parse the order of the stages (and which services implement them)
     order = [split(x, STAGE_SEP) for x in split(config.get(section, 'order'))]
 
+
+    #
+    # Browse through the list of (service, stage) Tuple to skip, if any
+    #
+    if skip_stages is not None:
+        # Check the types of skip_Stages. must be a list of Tuples
+        assert isinstance(skip_stages,list) and isinstance(skip_stages[0],tuple), 'ERROR: skip_stages must be a list of Tuples'
+
+        # reformat order into a list of tuples
+        order = [(x[0],x[1]) for x in order]
```

Comment:
space after comma; all you have to do here is:
```python
order = [(x[0], x[1]) for x in order if (x[0], x[1]) not in skip_services]
```

Reply:
I would rather write:

`order = [[x[0],x[1]] for x in order if (x[0],x[1]) not in skip_services]` so that `order` keeps its original format of a list of list

Reply:
That's fine, but then there's no reason (and in fact it's inconsistent) to construct the list of tuples up above (as I had done). So make `skip_services` into a list of 2-lists as well anywhere that is done, and I'm fine with this. (but space after a comma)